src/Services/ConvertToQtiService.php

```diff
@@ -38,15 +38,13 @@ class ConvertToQtiService
     protected $shouldAppendLogs           = false;
     protected $shouldGuessItemScoringType = true;
     protected $shouldUseManifest          = true;
-
```

Comment:
Formatting: A lot of unnecessary whitespace changes in this file. This is likely to cause unnecessary merge conflicts, and doesn't improve readability. Can these be removed if they are not needed?

Reply:
Done